plugins/module_utils/omevv_utils/omevv_firmware_utils.py

```diff
@@ -626,3 +630,36 @@ def delete_baseline_profile(self, profile_id, vcenter_uuid):
         resp = self.omevv.invoke_request(
             "DELETE", BASELINE_PROFILE_URI.format(vcenter_uuid=vcenter_uuid) + "/" + str(profile_id))
         return resp
+
+
+class OMEVVFirmwareUpdate:
+    def __init__(self, omevv):
+        self.omevv = omevv
+
+    def update_cluster(self, payload, vcenter_uuid, cluster_group_id):
```

Comment:
provide all payload values as parameter in function and return after converting to json_data applies to all the places

Reply:
Incorporated